### PR TITLE
Update Vault plugin create vault example, outdated token 

### PR DESCRIPTION
**Pull request by tlblessing**
https://konghq.atlassian.net/browse/FTI-2082
https://konghq.atlassian.net/browse/DOCS-1436

Direct review link:

https://deploy-preview-2467--kongdocs.netlify.app/hub/kong-inc/vault-auth/#create-a-vault

This plugin is not on the latest Kong Manager UI https://manager-reference.kong-cloud.com/default/plugins/select/.

### CODE REVIEW

**Comment by lena-larionova** [comment]
> This plugin is not on the latest Kong Manager UI https://manager-reference.kong-cloud.com/default/plugins/select/.

This is a good point - I imagine this is something like the Collector plugin for Brain/Immunity, where you need to perform some extra steps to set it up. The other possibility is that it's a separately licensed plugin - but I can't find any info on that.

File: app/_hub/kong-inc/vault-auth/index.md [hunk 1]
```diff
@@ -108,25 +108,27 @@ A Vault object represents the connection between Kong and a Vault server. It def
...
 ```bash
-$ curl -X POST http://kong:8001/vaults \
-  --data name=kong-auth \
-  --data mount=kong-auth \
-  --data protocol=http \
-  --data host=127.0.0.1 \
-  --data port=8200
-  --data token=s.m3w9gdV0uMDYFpMgEWSB2mtM
+$ curl --request POST \
+  --url http://localhost:8001/vaults \
```
Comment (lena-larionova) [inline app/_hub/kong-inc/vault-auth/index.md:112]:
```suggestion
$ curl -X POST http://localhost:8001/vaults \
```
`-X` is the shorthand for `--request`. Just changing this to the standard style we tend to use across the docs (tested, this does work).

File: app/_hub/kong-inc/vault-auth/index.md [hunk 2]
```diff
@@ -108,25 +108,27 @@ A Vault object represents the connection between Kong and a Vault server. It def
...
+  --header 'Content-Type: multipart/form-data' \
+  --form name=kong-auth \
+  --form mount=kong-auth \
+  --form protocol=http \
+  --form host=127.0.0.1 \
+  --form port=8200 \
+  --form vault_token=<token>
 HTTP/1.1 201 Created
```
Comment (lena-larionova) [inline app/_hub/kong-inc/vault-auth/index.md:120]: I'd suggest splitting out the request and the result into two codeblocks. That way, someone can click the copy icon to copy the request only.

Comment (tlblessing) [reply app/_hub/kong-inc/vault-auth/index.md:122]: Done in commit 20002d3

**Review by lena-larionova — APPROVED** [review]
LGTM!

**Pull request merged**